GameProject/Assets/Turn based assets/Scripts/MouseSelection.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using UnityEngine;
+
+namespace Turn_based_assets.Scripts
+{
+    public class MouseSelection : MonoBehaviour
+    {
+        private ISelection _selection;
+        [SerializeField] private PlayerController player;
+        public Vector3 rawGridPoint;
+        public float distanceY;
+        public float offset;
+        public float gridSize;
+        public Plane _plane;
```

Comment:
There's quite a few varying conventions going on here - I haven't seen a conventions file or anything pointing towards how you name public and private fields in this project, but switching between prefixing public fields and private fields with `_` seems confusing, maybe normalizing it would be a good start - especially for publicly exposed fields.

Reply:
Yeah this seems like another case of duped work